src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -13,6 +13,7 @@
 
 #endregion
 
+#if !UNIX
```

Comment:
`Unblock-File` is excluded from Unix at all so we don't need to exclude the code here.